internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -669,6 +681,16 @@ func (s *LoadBalancedWebService) convertNetworkLoadBalancer() (networkLoadBalanc
 	return config, nil
 }
 
+func (s *LoadBalancedWebService) convertGracePeriod() *int64 {
+	var gracePeriod *int64 = aws.Int64(int64(manifest.DefaultHealthCheckGracePeriod))
+	if s.manifest.HTTPOrBool.Main.HealthCheck.Advanced.GracePeriod != nil {
+		gracePeriod = aws.Int64(int64(s.manifest.HTTPOrBool.Main.HealthCheck.Advanced.GracePeriod.Seconds()))
+	} else if s.manifest.NLBConfig.Listener.HealthCheck.GracePeriod != nil {
```

Comment:
i assume we can't set both alb and nlb grace period at the same time. Maybe `if` is enough?

Reply:
So we either want to set grace period that is given in ALB or NLB or default. So if we just use `if` then we wouldn't be able to send default value if both ALB and NLB doesn't have `grace_period` specified. Does that make sense?

Reply:
nit:  consider switching the layout to guard-clauses

```go
if s.manifest.HTTPOrBool.Main.HealthCheck.Advanced.GracePeriod != nil {
   return aws.Int64(int64(s.manifest.HTTPOrBool.Main.HealthCheck.Advanced.GracePeriod.Seconds()))
}
if s.manifest.NLBConfig.Listener.HealthCheck.GracePeriod != nil {
   return aws.Int64(int64(s.manifest.NLBConfig.Listener.HealthCheck.GracePeriod.Seconds()))
}
return aws.Int64(int64(manifest.DefaultHealthCheckGracePeriod))
```